pkg/commands/docker.go

```diff
@@ -53,13 +53,14 @@ func (u *dockerUsernamePassword) Authenticate(runner build.Runner) error {
 
 // NewDockerBuild creates a build target with specified docker file and build parameters
 func NewDockerBuild(dockerfile, contextDir string,
-	buildArgs, buildSecretArgs []string, registry, imageName string, pull, noCache bool) build.Target {
+	buildArgs, buildSecretArgs []string, registry string, imageNames []string, pull, noCache bool) build.Target {
+
+	var pushTo []string
 
-	var pushTo string
 	// If imageName is empty, skip push.
 	// If registry is empty, it means push to DockerHub.
-	if len(imageName) > 0 {
-		pushTo = fmt.Sprintf("%s%s", registry, imageName)
+	for _, imageName := range imageNames {
```

Comment:
>imageName [](start = 8, length = 9)

Corner case: will it fail?  --docker-registry myregistry.azurecr.io -t ""

Reply:
Yep, good catch. I updated `main.go` with a change so that string slices can't be == `""` - it will skip appending them to the array. This also affected build envs (although I didn't check if it would error there), so a double fix!